neofs_config.mk

```diff
@@ -21,3 +21,7 @@ update.eigen_trust_iterations:
 # Update basic income rate in fixed 12 (make update.basic_income_rate val=1000)
 update.basic_income_rate:
 	@./bin/config.sh BasicIncomeRate $(val)
+
+# Tick epoch (nonsense is passed as arguments to provoke an epoch change without changing the script)
```

Comment:
This string used in help output so it should not mention implementation details. Keep it short, e.g. `Tick new epoch`. 

As for details about `k 0`, we can mention it down there. I would suggest something like that:

```Makefile
# Tick new epoch in side chain
tick.epoch:
#       UnusedGlobalVariable is passed as argument to 
#       provoke epoch change without any changes in `bin/*`. 
 	@./bin/config.sh UnusedGlobalVariable 0
```

Reply:
Done